PSA/deployVM.ps1

```diff
@@ -0,0 +1,93 @@
+# Adapted from https://github.com/weeyin83/vm-actions
```

Comment:
Shall we move this file to a separate subdirectory ?

either `scripts` or, if it's just for CI purposes (don't remember sry), under `.github/vms` for example?

Reply:
moved to `.github/azure`.